dags/operators/email_schema_change_operator.py

```diff
@@ -0,0 +1,69 @@
+import boto3
+import logging
+import re
+from airflow.exceptions import AirflowException
+from airflow.operators.email_operator import EmailOperator
+from airflow.utils.decorators import apply_defaults
+from difflib import unified_diff
+from os import environ
+
+
+class EmailSchemaChangeOperator(EmailOperator):
+    """
+    Execute a Spark job on EMR and send email on schema changes.
+
+    :param latest_schema: The name of the job.
+    :type latest_schema: string
+
+    :param previous_schema: The e-mail address of the user owning the job.
+    :type previous_schema: string
+
+    and all EmailOperator params except html_content
+    """
+    template_fields = EmailOperator.template_fields + ('latest_schema', 'previous_schema')
+    s3_regex = re.compile('([Ss]3://)?(?P<Bucket>[^/]*)/(?P<Key>.+)')
```

Comment:
Any particular reason to do this vs passing in bucket/key/dates as separate params to begin with?